wp-content/plugins/taste-graph-connector/admin/settings.php

```
@@ -0,0 +1,696 @@
+    exit;
+}
+
+/**
+ * TGC_Admin_Settings class
+ */
+class TGC_Admin_Settings {
+    
+    /**
+     * Settings page hook suffix
+     */
+    private static $hook_suffix = '';
```

PHPMD (2.15.0)

Avoid unused private fields such as '$hook_suffix'. (Unused Code Rules)

(UnusedPrivateField)